python/pyspark/sql/functions.py

```diff
@@ -1798,6 +1798,22 @@ def create_map(*cols):
     return Column(jc)
 
 
+@ignore_unicode_prefix
+@since(2.4)
+def create_map_fromarray(col1, col2):
```

Comment:
What about calling it just `map_from_arrays`?

Reply:
Thanks, I would like to hear other opinions, too.

Reply:
cc: @gatorsmile @ueshin